.github/workflows/ui-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install jupyter_packaging jupyterlab
+          python -m pip install jupyter_packaging "jupyterlab==4.0.12"
```

Comment:
What is the reason for pinning on a particular version?

Should there be a comment to explain why, when we look at it again in the future?

Reply:
JupyterLab 4.1 is breaking galata tests here and there.

Reply:
https://github.com/voila-dashboards/voila-gridstack/issues/220

Reply:
What does "breaking" mean? Is it snapshots mismatch? Galata runtime error? Something else?

Reply:
Galata runtime error

Reply:
I opted for a quick solution to unblock the releaser.

Reply:
Sounds good to me. Maintaining the tooling around JupyterLab plugins can be painful, and it should not go in the way of getting things done.

Let's debug this separately.

Reply:
Sure thanks.

Would you mind posting more information about that "Galata timeout", here or in #220?

Wondering whether this should be fixed upstream in JupyterLab. I also noticed some changes related to the use of Galata downstream (for example in https://github.com/jupyterlite/jupyterlite/pull/1263), so it could be useful to have this information somewhere.

Reply:
All the await calls to click on an element have the timeout error.

Reply:
Bumping Galata to 5.1.0 fixes the issue